lib/arsi/relation.rb

```diff
@@ -1,6 +1,6 @@
 module Arsi
   module Relation
-    attr_accessor :without_arsi
+    attr_writer  :without_arsi
```

Comment:
Am I curious why this change? We won't need to read this attribute anymore?

Reply:
Do we need a spec for this kind of change?

Reply:
It's defined just below this. I was getting a warning about redefinition of the reader method because we had `attr_accessor :without_arsi` followed immediately by `def with_out_arsi`